[rospy.client][INFO] 2017-06-09 18:04:20,639: init_node, name[/move_arm_11666_1497056660593], pid[11666]
[xmlrpc][INFO] 2017-06-09 18:04:20,639: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2017-06-09 18:04:20,643: Started XML-RPC server [http://ip-172-31-19-100:34779/]
[rospy.init][INFO] 2017-06-09 18:04:20,644: ROS Slave URI: [http://ip-172-31-19-100:34779/]
[rospy.impl.masterslave][INFO] 2017-06-09 18:04:20,647: _ready: http://ip-172-31-19-100:34779/
[xmlrpc][INFO] 2017-06-09 18:04:20,656: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2017-06-09 18:04:20,656: Registering with master node http://localhost:52701
[rospy.init][INFO] 2017-06-09 18:04:20,751: registered with master
[rospy.rosout][INFO] 2017-06-09 18:04:20,755: initializing /rosout core topic
[rospy.rosout][INFO] 2017-06-09 18:04:20,765: connected to core topic /rosout
[rospy.simtime][INFO] 2017-06-09 18:04:20,770: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2017-06-09 18:04:21,002: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2017-06-09 18:04:21,003: topic[/motion_planning_goal] adding connection to [http://ip-172-31-19-100:37719/], count 0
[rospy.internal][INFO] 2017-06-09 18:04:21,005: topic[/joint_states] adding connection to [http://ip-172-31-19-100:38886/], count 0
[rospy.internal][INFO] 2017-06-09 18:04:21,007: topic[/joint_trajectory] adding connection to [/marker_control_9450_1497056458652], count 0
[rospy.internal][INFO] 2017-06-09 18:04:21,881: topic[/joint_trajectory] adding connection to [/robot_sim], count 1
[rospy.internal][INFO] 2017-06-09 18:04:22,002: topic[/collision_object] adding connection to [/move_group], count 0
[rospy.internal][INFO] 2017-06-09 18:04:22,006: topic[/attached_collision_object] adding connection to [/move_group], count 0
[rosout][ERROR] 2017-06-09 18:04:40,437: bad callback: <bound method MoveArm.move_arm_cb of <__main__.MoveArm object at 0x7fee718ae390>>
Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/topics.py", line 720, in _invoke_callback
    cb(msg)
  File "/home/ccc_v1_w_dd4cc_18220/asn13529_Project_5/asn13530_Project_4/work/catkin_ws/src/motion_planning/scripts/motion_planning.py", line 326, in move_arm_cb
    trajectory = self.project_plan(numpy.array(q_start), q_goal, self.q_min, self.q_max)
  File "/home/ccc_v1_w_dd4cc_18220/asn13529_Project_5/asn13530_Project_4/work/catkin_ws/src/motion_planning/scripts/motion_planning.py", line 311, in project_plan
    q_list = self.motion_plan(q_start, q_goal, q_min, q_max)
  File "/home/ccc_v1_w_dd4cc_18220/asn13529_Project_5/asn13530_Project_4/work/catkin_ws/src/motion_planning/scripts/motion_planning.py", line 259, in motion_plan
    point = self.get_sample_point(q_min, q_max)
  File "/home/ccc_v1_w_dd4cc_18220/asn13529_Project_5/asn13530_Project_4/work/catkin_ws/src/motion_planning/scripts/motion_planning.py", line 209, in get_sample_point
    point.append(random.unifrom(q_min[i], q_max[i]))
AttributeError: 'module' object has no attribute 'unifrom'

[rospy.core][INFO] 2017-06-09 18:05:09,895: signal_shutdown [signal-2]
[rospy.internal][INFO] 2017-06-09 18:05:09,926: topic[/collision_object] removing connection to /move_group
[rospy.internal][INFO] 2017-06-09 18:05:09,930: topic[/joint_trajectory] removing connection to /marker_control_9450_1497056458652
[rospy.internal][INFO] 2017-06-09 18:05:09,934: topic[/joint_trajectory] removing connection to /robot_sim
[rospy.internal][INFO] 2017-06-09 18:05:09,937: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2017-06-09 18:05:09,941: topic[/attached_collision_object] removing connection to /move_group
[rospy.internal][INFO] 2017-06-09 18:05:09,947: topic[/motion_planning_goal] removing connection to http://ip-172-31-19-100:37719/
[rospy.internal][WARNING] 2017-06-09 18:05:09,947: Unknown error initiating TCP/IP socket to ip-172-31-19-100:47916 (http://ip-172-31-19-100:37719/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.local_endpoint = self.socket.getsockname()
AttributeError: 'NoneType' object has no attribute 'getsockname'

[rospy.internal][INFO] 2017-06-09 18:05:09,951: topic[/joint_states] removing connection to http://ip-172-31-19-100:38886/
[rospy.impl.masterslave][INFO] 2017-06-09 18:05:09,959: signal-2
[rospy.core][INFO] 2017-06-09 18:05:10,528: signal_shutdown [atexit]
